src/Shared/Components/TagsKeyValueTable/utils.tsx

```diff
@@ -0,0 +1,23 @@
+import { DynamicDataTableRowDataType, DynamicDataTableRowType } from '../DynamicDataTable'
+import { TagsTableColumnsType } from './types'
+
+export const getEmptyTagTableRow = (): DynamicDataTableRowType<TagsTableColumnsType> => ({
```

Comment:
this is being used in dashboard and fe-lib check if this can be used there as well

Reply:
importing in dashboard from common, but have different in fe-lib